Retail and SLE15 Images update (#303)
uyuni-project/uyuni-docs: modules/administration/pages/image-management.adoc

+
image::os-image-build-host.png[scaledwidth=80%]
+

> . Navigate to the menu:System Details[Software > Software Channels] tab, enable required software channels depending on the build host version.

Reviewer: ```suggestion
. Navigate to menu:System Details[Software > Software Channels], and enable the required software channels depending on the build host version.
```

. To schedule a build, click the btn:[Build] button.

[IMPORTANT]
====

> Build server cannot run any form of automounter during the image building. Make sure Gnome session is not running under root user if used.
> 
> If automounter is running, image build will finish successfully, however checksum of image will be different and will fail later.

Reviewer: ```suggestion
The build server cannot run any form of automounter during the image building process. 
If applicable, ensure that you do not have your Gnome session running as root.
If an automounter is running, the image build will finish successfully, but the checksum of the image will be different and will fail later.
```
